hydra-node/src/Hydra/Chain/Direct.hs

```diff
@@ -215,11 +232,11 @@ withDirectChain tracer networkId iocp socketPath keyPair party cardanoKeys point
 -- current point in time.
 queryTimeHandle :: MonadThrow m => NetworkId -> FilePath -> IO (TimeHandle m)
 queryTimeHandle networkId socketPath = do
-  systemStart <- querySystemStart networkId socketPath
-  eraHistory <- queryEraHistory networkId socketPath
+  tip@(ChainPoint slotNo _) <- queryTip networkId socketPath
+  systemStart <- querySystemStart networkId socketPath (QueryAt tip)
+  eraHistory <- queryEraHistory networkId socketPath (QueryAt tip)
```

Comment:
This is probably how we should be querying most things generally speaking -> first get a point, and then run all subsequent queries on that point. It's nothing really crucial though because we explicitly said Hydra heads would need to be terminated in case of protocol parameter updates thus in principle, the case where we get different parameters across several nearby queries does not happen. 
Still.. I'd prefer doing things properly from the get-go, so that the day we decide to do something more elaborate regarding hard-forks or protocol updates, we have one less problem to think of.